queries.sql

```diff
@@ -1,3 +1,145 @@
 /*Queries that provide answers to the questions from all projects.*/
 
```

Comment:
- [ ] Well done guys. But in your pull request description, I see you have included only one screenshot for each problematic query. It should be screenshots for **before** and **after** you improve the speed for each of the 3 problematic queries. Please fix this.
- [ ] Secondly, in your schema file, I should see a solution that you have implemented. Something like: 

```sql
   CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);
```
So please after you insert the data as you have already done. Run each problematic query and take the **before** screenshot of the result and then execute the above code as a fix. Afterward, run the query again and take the **after** screenshot.

Reply:
@bobb-ROd
Snice I have already inserted the data should I remove it now??

Reply:
Resolved